src/Serilog/NewRelic.LogEnrichers.Serilog.Tests/FormatterTests.cs

```diff
@@ -11,7 +11,7 @@ namespace NewRelic.LogEnrichers.Serilog.Tests
     public class FormatterTests
     {
         private const string LinkingMetadataKey = "newrelic.linkingmetadata";
-        private const string UserPropertyKeyPrefix = "Message Properties.";
+        private const string UserPropertyKeyPrefix = LoggingExtensions.UserPropertyPrefix;
```

Comment:
I am not sure what we should do here.  Part of me thinks it should remain as a const string in the test to ensure that we don't introduce an inadvertent change to the helper class without considering the ramifications?